web/themes/custom/westy/assets/css/westy.style.css

```diff
@@ -1 +1,12300 @@
+@charset "UTF-8";
```

Comment:
You need to `npm run production` to get the minified versions of CSS and JS.

Reply:
ran the command to minify css and js